docs/UserGuide.md

```diff
@@ -419,15 +555,15 @@ Optional identifiers can be added to show the list of matched clients with only
 <br><br>
```

Comment:
The link for the `Return to Table of Contents` is somehow indented, when it should be on the outermost indent level. Can you see if you can resolve it?

Reply:
Fixed.